puppet/modules/profiles/manifests/foreman.pp

```diff
@@ -11,4 +11,26 @@
   puppet::config::main { 'dns_alt_names':
     value => $foreman::serveraliases,
   }
+
+  package {'rubygem-foreman_maintain':
+    ensure => present,
+  }
+
+  $backup_base_path = '/var/backups'
+  $backup_path = "${backup_base_path}/foreman"
+
+  file { $backup_base_path:
+    ensure => directory,
+  }
+  file { $backup_path:
+    ensure => directory,
+  }
+
+  include profiles::backup::sender
+
+  restic::repository { 'foreman':
+    backup_cap_dac_read_search => true,
+    backup_path                => $backup_path,
+    backup_pre_cmd             => ["+/usr/bin/foreman-maintain backup online --assumeyes --preserve-directory ${backup_path}"],
```

Comment:
https://www.freedesktop.org/software/systemd/man/systemd.service.html#Command%20lines is neat

Reply:
I did not know systemd could do this, but nice!

Reply:
Neither did I until a few days ago :)